hazelcast/src/main/java/com/hazelcast/internal/management/dto/ClusterHotRestartStatusDTO.java

```diff
@@ -125,4 +131,26 @@ public void fromJson(JsonObject root) {
             memberHotRestartStatusMap.put(member, status);
         }
     }
+
+    @Override
+    @SuppressWarnings("checkstyle:innerassignment")
+    public boolean equals(Object obj) {
```

Comment:
If you don't need `equals()` and `hashCode()` in production, just for an `assertEquals()` it's mostly easier to write a specialized assert method in the test than to implement those methods. The main reason is that they normally decrease the code coverage due to the missing branch coverage (especially of the `equals()` method).

An example for this are some of the config class tests, which just compare the fields instead of implementing `equals()` and `hashCode()`.